pethublocal/functions.py

```diff
@@ -1006,9 +1008,11 @@ def parse_mqtt_message(pethubconfig, mqtt_topic, mqtt_message):
                                     animals.append(pets[tag]['Name'])  # Add name to Payload
                                     # Update config with current state
                                     if 'PetMovement' in mqttmessage.Operation:
+                                        DeviceMac = str(mqttmessage.Update_Item[0])[13:30]
```

Comment:
Wouldn't using the `mqtt_topic_split` be a better option as I think:
```
DeviceMac = str(mqtt_topic_split[-1])
```
Gives you the last value in the topic, which is the device Mac Address

Reply:
Works like a charm, Peter! Thanks, I did not know about it!